packages/forklift-console-plugin/src/modules/Plans/views/list/components/MigrationTypeCell.tsx

```diff
@@ -1,36 +1,53 @@
 import React, { FC } from 'react';
+import { DateTime } from 'luxon';
+import { usePlanMigration } from 'src/modules/Plans/hooks/usePlanMigration';
 import { PlanCutoverMigrationModal } from 'src/modules/Plans/modals';
 import { isPlanArchived, isPlanExecuting } from 'src/modules/Plans/utils';
 import { useModal } from 'src/modules/Providers/modals';
 import { useForkliftTranslation } from 'src/utils';
 
-import { Button, ButtonVariant, Flex, Label } from '@patternfly/react-core';
+import { Button, ButtonVariant, Flex, Label, Tooltip } from '@patternfly/react-core';
 
 import { CellProps } from './CellProps';
 
 export const MigrationTypeCell: FC<CellProps> = ({ data }) => {
   const { t } = useForkliftTranslation();
   const { showModal } = useModal();
   const plan = data?.obj;
+  const [lastMigration, migrationLoaded] = usePlanMigration(plan);
 
-  const isWarmAndExecuting = plan?.spec?.warm && isPlanExecuting(plan);
-  const isArchived = isPlanArchived(plan);
+  const isWarmAndExecuting = plan?.spec?.warm && isPlanExecuting(plan) && !isPlanArchived(plan);
 
   const onClickPlanCutoverMigration = () => {
     showModal(<PlanCutoverMigrationModal resource={plan} />);
   };
 
+  const cutoverSet = Boolean(lastMigration?.spec?.cutover);
```

Comment:
This part also repeat itself a few times, consider extracting it to a const/util func